### PR TITLE
Add privacy manifest

### PR DESCRIPTION
**Pull request by KantaSwift**
## Summary
To comply with Apple's new privacy policies every third-party SDK should include a Privacy manifest in case if they access APIs which potentially can be used for fingerprinting.
So, I believe it falls under the following content, I add the PrivacyInfo.xcprivacy

- [C56D.1 ](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api)

### CODE REVIEW

**Review by Ryu0118 — COMMENTED** [review]
@KantaSwift これもっす

**Comment by KantaSwift** [comment]
@Ryu0118 
同様にターゲットごとに追加していく方針で対応した

File: .swiftpm/xcode/xcshareddata/xcschemes/Mugen.xcscheme [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+         </BuildActionEntry>
+         <BuildActionEntry
+            buildForTesting = "YES"
+            buildForRunning = "YES"
+            buildForProfiling = "NO"
+            buildForArchiving = "NO"
+            buildForAnalyzing = "YES">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "MugenTests"
```
Comment (Ryu0118) [inline .swiftpm/xcode/xcshareddata/xcschemes/Mugen.xcscheme]: お手数なんですが、.swiftpmをgitignoreにしてほしいっす

**Review by Ryu0118 — CHANGES_REQUESTED** [review]

**Review by Ryu0118 — APPROVED** [review]
LGTM

**Pull request merged**